src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.26.0
+
+11 November 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- **Hostname Length Limit**: Deploys now fail if a service hostname exceeds 63 characters, showing an error message. Previously, the dev environment deployed successfully even if the endpoint didn’t work. This may affect environments that deployed without issues before.
+- **Helm Release Name Limit**: Helm release names are now limited in length. Use defaultBackend.nameOverride to shorten the default backend service name if needed. If changing this during an upgrade, [follow the guide as it may impact the installation](self-hosted/helm-configuration.mdx#manual-migration-steps-when-renaming-the-defaultbackend-service).
```

Comment:
they can also use `fullnameOverride` or `nameOverride`, not necessarily have to rename the backend